jax_unirep/utils.py

```diff
@@ -181,6 +207,26 @@ def load_params_1900(name: str = "uniref50") -> Dict:
     return params
 
 
+def load_params(dir_path: str) -> Dict:
+    """Load pre-trained mLSTM1900 weights from any JAX-unirep outputted source."""
+
+    params = dict()
+
+    params["gh"] = np.load(dir_path + "/rnn_mlstm_mlstm_gh/0.npy")
```

Comment:
The character before `0` should be a colon. It’s only in macOS Finder that the colon gets rendered as a `/`.